package.json

```diff
@@ -89,7 +90,7 @@
     "rimraf": "^2.6.1",
     "shelljs": "^0.8.2",
     "stylelint": "^9.0.0",
-    "stylelint-config-terra": "^1.0.0",
+    "stylelint-config-terra": "git+https://github.com/cerner/stylelint-config-terra.git#Errors",
```

Comment:
We have somewhat of a chicken and the egg problem with the linter. We want the lint rules turned on for this pull request and immediately on when this gets merged into master.

A couple options that come to mind.

1. Release a major version of stylelint-config-terra with the rules turned on to throw errors.
2. Revert this line back to "stylelint-config-terra": "^1.0.0" and temporarily turn on the rules in terra-core manually. You can override the stylelint rules in the package.json or create a separate stylelint.config.js file with the rules turned on. It looks like this: https://github.com/cerner/stylelint-config-terra/blob/Errors/stylelint.config.js#L24-L28

Reply:
I reverted back to the release version and turned on our stylelint rules. Once we release styelint-config-terra, I'll remove the rules